### PR TITLE
Update BrowserWindow 

### PR DESCRIPTION
**Pull request by tong**

### CODE REVIEW

File: src/electron/main/BrowserWindow.hx [hunk 1]
```diff
@@ -25,6 +26,15 @@ import haxe.Constraints.Function;
     var app_command : BrowserWindowEvent<String->Void> = "app-command";
 }
 
+@:enum abstract ThumbarButtosFlag(String) from String to String {
```
Comment (fponticelli) [inline src/electron/main/BrowserWindow.hx]: is this correct?

File: src/electron/main/BrowserWindowOptions.hx [hunk 2]
```diff
@@ -37,33 +45,35 @@ typedef BrowserWindowOptions = {
...
+    @:optional var hasShadow : Bool;
+    @:optional var darkTheme : Bool;
     @:optional var transparent : Bool;
     @:optional var type : WindowType;
-    @:native("standard-window") @:optional var standard_window : Bool;
-    @:native("title-bar-style") @:optional var title_bar_style : Bool;
-    @:native("web-preferences") @:optional var web_preferences : WebPreferences;
+    @:optional var titleBarStyle : TitleBarStyle;
+	@:optional var webPreferences : WebPreferences;
 }
```
Comment (fponticelli) [inline src/electron/main/BrowserWindowOptions.hx:108]: indentation in general is all over the place ;)

File: src/electron/main/BrowserWindow.hx (hunk 1 above)
Comment (tong) [inline src/electron/main/BrowserWindow.hx]: Except of the missing 'n' yes, it's a windows thing :)

File: src/electron/main/BrowserWindowOptions.hx (hunk 2 above)
Comment (tong) [inline src/electron/main/BrowserWindowOptions.hx:108]: Damn, wondering why my editor tab settings have changed (?)

**Comment by tong** [comment]
Sorry for the mess. Always to lazy for branching. Hope it's fixed now.

**Comment by fponticelli** [comment]
Looks great, thank you so much!

**Comment by tong** [comment]
Thank you!

**Pull request merged**